src/components/FeaturePanel/ImageSection/Photo.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Image } from '../../../services/types';
+
+interface PhotoProps {
+  image: Image;
+}
+
+export const Photo: React.FC<PhotoProps> = ({ image }) => {
+  if (image.isPano) {
+    const pannellumUrl = `https://cdn.pannellum.org/2.5/pannellum.htm#panorama=${encodeURIComponent(
+      image.thumb,
+    )}&autoLoad=true`;
```

Comment:
I think the panellum controls are not necessary, can you hide them? It kinda collides with the map controls (+/-) and pano is not the primary information of the page, so we don't want the user to be distracted that much 🙂 

Instead we can perhaps add `autoRotate=-1` which helps users find that it is movable.

Reply:
Fair points, but I personally think that the full screen button is quite nice as there is no other way to get this fully immersive view without leaving osmapp and going to mapilarry itself. What do you think of keeping the full screen button?

Reply:
Ok, i agree. Lets leave it now, until there is another way of enlarging it. Good point! 👍 

Can you try to remove the +/-, then? If it is not possible, it is not a blocker :)